Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/TwoQubitRB.py

```diff
@@ -245,3 +267,40 @@ def run(
             num_averages=num_shots_per_circuit,
             state=job.result_handles.get("state").fetch_all(),
         )
+
```

Comment:
I think we should remove the example section line 243 above, don't you think?

Reply:
Agreed. Done